server/models/message.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require('mongoose');
+
+const messageSchema = new mongoose.Schema({
+    text: {
+        type: String,
+        required: true
+    },
+    senderId: {
+        type: mongoose.Types.ObjectId,
+        ref: 'User',
+        required: true
+    },
+    conversationId: {
+        type: mongoose.Types.ObjectId,
+        ref: 'Conversation',
+        required: true
+    },
+
+},
+    {
+        timestamps: true
+    });
```

Comment:
Please let's use the recommended linters setup in our editor to format this code

Reply:
Done the changes